pvlib/test/test_bifacial.py

```diff
@@ -31,38 +32,38 @@ def test_pvfactors_timeseries():
     horizon_band_angle = 15.
 
     # Expected values
-    expected_ipoa_front = pd.Series([1034.96216923, 795.4423259],
+    expected_ipoa_front = pd.Series([1034.95474708997, 795.4423259036623],
                                     index=timestamps,
-                                    name=(1, 'front', 'qinc'))
-    expected_ipoa_back = pd.Series([92.11871485, 70.39404124],
+                                    name=('total_inc_front'))
+    expected_ipoa_back = pd.Series([91.88707460262768, 78.05831585685215],
                                    index=timestamps,
-                                   name=(1, 'back', 'qinc'))
+                                   name=('total_inc_back'))
 
     # Test serial calculations
-    ipoa_front, ipoa_back, df_registries = pvfactors_timeseries(
+    ipoa_front, ipoa_back = pvfactors_timeseries(
         solar_azimuth, solar_zenith, surface_azimuth, surface_tilt,
+        axis_azimuth,
         timestamps, dni, dhi, gcr, pvrow_height, pvrow_width, albedo,
         n_pvrows=n_pvrows, index_observed_pvrow=index_observed_pvrow,
         rho_front_pvrow=rho_front_pvrow, rho_back_pvrow=rho_back_pvrow,
         horizon_band_angle=horizon_band_angle,
-        run_parallel_calculations=False, n_workers_for_parallel_calcs=None)
+        run_parallel_calculations=False, n_workers_for_parallel_calcs=-1)
 
     pd.testing.assert_series_equal(ipoa_front, expected_ipoa_front)
     pd.testing.assert_series_equal(ipoa_back, expected_ipoa_back)
-    pd.testing.assert_index_equal(timestamps, df_registries.index.unique())
 
     # Run calculations in parallel
-    ipoa_front, ipoa_back, df_registries = pvfactors_timeseries(
+    ipoa_front, ipoa_back = pvfactors_timeseries(
```

Comment:
@anomam I suggest splitting the `run_parallel_calculations=False|True` into two tests. Maybe something like this (including optional `xfail` if you don't know the reason for windows failures).

```python
from conftest import platform_is_windows

@pytest.mark.parametrize('run_parallel_calculations', [
    False,
    pytest.param(True, marks=pytest.mark.xfail(platform_is_windows)
])
def test_pvfactors_timeseries(run_parallel_calculations):
    # modify test as needed for arg
```